README.md

```diff
@@ -59,12 +59,11 @@ Overload processing: Task queue mechanism, the number of schedulable tasks on a
 
 
 ### Recent R&D plan
-Work plan of Dolphin Scheduler: [R&D plan](https://github.com/apache/incubator-dolphinscheduler/projects/1), which `In Develop` card shows the features that are currently being developed and TODO card means what needs to be done(including feature ideas).
+The work plan of Dolphin Scheduler: [R&D plan](https://github.com/apache/incubator-dolphinscheduler/projects/1), which `In Develop` card shows the features that are currently being developed and TODO card lists what needs to be done(including feature ideas).
```

Comment:
The work plan of Dolphin Scheduler: [R&D plan](https://github.com/apache/incubator-dolphinscheduler/projects/1), in which the `Doing` column shows the features are currently being developed and the `TODO` column lists what needs to be done (including feature ideas).

Reply:
Hi, I thought "develop" here is considered as a transitive verb. I keep the idea of not add the preposition "in" since the precondition of placing the preposition before the which clause instead of the end of a sentence is the preposition exists. Whats' your opinion here?